app/views/quotes/_quote.html.erb

```diff
@@ -1 +1,7 @@
-<div><%= quote.author %> said, <q><%= quote.body %></q>, in the <%= quote.location %></div>
+<div>
+  <%= quote.author %> said, <q><%= quote.body %></q>, in the <%= quote.location %>
+  <% if quote.user == current_user %>
```

Comment:
This is an authorization concern. What if I want a group of people to be able to remove any quote? but not everyone? I think we should remove this check for now. Or we should an authorization solution like CanCanCan or Pundit.